firestore/google/cloud/firestore_v1beta1/proto/common.proto

```diff
@@ -17,8 +17,8 @@ syntax = "proto3";
 
 package google.firestore.v1beta1;
 
-import "google/api/annotations.proto";
 import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
```

Comment:
@busunkim96 This PR seems to be de-linting the protos.  It looks like the change is not within our process, though (this version shows up in the upstream `googleapis` repo).